jest.conf/testUtils.js

```diff
@@ -29,3 +22,56 @@ export const testAfterResize = testFunction => {
   };
   animationFrameId = requestAnimationFrame(assertAfterResize);
 };
+
+export async function renderComponent(page, component, props) {
+  await page.evaluate(
+    ({ component, props }) => {
+      window.postMessage(
+        {
+          type: 'RENDER_COMPONENT',
+          component: component,
+          props: props,
+        },
+        '*'
+      );
+    },
+    { component, props }
+  );
+  await page.waitForSelector('#testing-playground');
+
+  const isComponentRendered = await page.evaluate(() => {
+    const testing_playground = document.querySelector('#testing-playground');
+    return testing_playground && testing_playground.children.length > 0;
+  });
+
+  if (!isComponentRendered) {
```

Comment:
I wonder if we could add something like `expect(isComponentRendered).toBe(true)` so this could be the assertion to know if a visual test pass. And if it dont, we could see more easily that an error is happening, because a console.error could go unnoticed in the console.

Reply:
Sure, will replace this check with the assertion.